jest.config.js

```diff
@@ -3,6 +3,7 @@ module.exports = {
   preset: 'ts-jest',
   testEnvironment: 'node',
   testMatch: ['**/src/test/unit/?(*.)+(spec|test).[jt]s?(x)'],
+  testPathIgnorePatterns: ['<rootDir>/tests/'],
```

Comment:
`testPathIgnorePatterns` defaults to `["/node_modules/"]`, let us keep it to avoid problems when adding more Jest unit tests.

```suggestion
  testPathIgnorePatterns: ['<rootDir>/tests/', '/node_modules/'],
```